src/components/views/TheHeader.vue

```diff
@@ -1,10 +1,22 @@
 <script>
 import { defineComponent } from 'vue';
 import TheContainer from '@/components/common/TheContainer.vue';
+import TopNavbar from '@/components/features/Header/TopNavbar.vue';
+import SignSection from '@/components/features/Header/SignSection.vue';
 
 export default defineComponent({
   components: {
     TheContainer,
+    TopNavbar,
+    SignSection,
+  },
+  computed: {
+    isPublicContent() {
+      return ['Home', 'Movies', 'Screenings', 'SingleMovie'].includes(this.$route.name);
```

Comment:
While not necessarily to be done in this PR - keeping such list (and the one below) here isn't really ideal.
There is a better way by utilizing the meta fields of the routes and then we would not need to care about the names or keeping them in lists.
https://router.vuejs.org/guide/advanced/meta.html